src/data/ability.ts

```diff
@@ -2739,9 +2739,9 @@ export class BlockOneHitKOAbAttr extends AbAttr {
   }
 }
 
-export class IncrementMovePriorityAbAttr extends AbAttr {
+export class ChangeMovePriorityAbAttr extends AbAttr {
   private moveIncrementFunc: (pokemon: Pokemon, move: Move) => boolean;
-  private increaseAmount: integer;
+  private increaseAmount: number;
```

Comment:
+ on @innerthunder 's comment
```suggestion
  private changeAmount: number;
```